example/image-fill-styles/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+      <title>wavesurfer.js | Image Fill Styles Example</title>
+      <link href="data:image/gif;" rel="icon" type="image/x-icon" />
+      <!-- Bootstrap -->
+      <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" rel="stylesheet">
+      <link rel="stylesheet" href="../css/style.css" />
+      <link rel="stylesheet" href="../css/ribbon.css" />
+      <link rel="screenshot" itemprop="screenshot" href="https://katspaugh.github.io/wavesurfer.js/example/screenshot.png" />
+      <!-- wavesurfer.js -->
+      <script src="../../dist/wavesurfer.js"></script>
+      <!-- Demo -->
+      <script src="main.js"></script>
+      <!-- highlight.js for syntax highlighting in this example -->
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.2.0/styles/default.min.css">
+      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.2.0/highlight.min.js"></script>
+      <script>hljs.highlightAll();</script>
+      <style>#waveform {
+         z-index:-5;
```

Comment:
indentation with 4 spaces. The spacing in example you're adding is also inconsistent.

Reply:
fixed